### PR TITLE
[WebProfiler] Removed an object as route generator argument

### PR DESCRIPTION
**Pull request by linaori**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | #17178 |
| License | MIT |
| Doc PR | ~ |

### CODE REVIEW

**Comment by stof** [comment]
:+1:

File: src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php [hunk 1]
```diff
@@ -307,7 +307,6 @@ public function searchResultsAction(Request $request, $token)
             'end' => $end,
             'limit' => $limit,
             'panel' => null,
-            'request' => $request,
```
Comment (iltar) [inline src/Symfony/Bundle/WebProfilerBundle/Controller/ProfilerController.php]: Not related to this issue but it was defined twice in the array.

**Comment by xabbuh** [comment]
:+1: 

Status: Reviewed

**Comment by stof** [comment]
Thank you @iltar.

**Pull request merged**